keras_retinanet/preprocessing/csv_generator.py

```diff
@@ -1,6 +1,4 @@
 """
-Copyright 2017-2018 Fizyr (https://fizyr.com)
```

Comment:
Is there anything we can add as a copyright statement? Your real name, or otherwise your github name perhaps?

Reply:
Or should it be Netra Inc. ?